shared/checks/oval/audit_rules_sysadmin_actions.xml

```diff
@@ -3,11 +3,20 @@
     <metadata>
       <title>Audit System Administrator Actions</title>
       <affected family="unix">
+{{% if product == "rhel6" %}}
+        <platform>Red Hat Enterprise Linux 6</platform>
+{{% else %}}
         <platform>Red Hat Enterprise Linux 7</platform>
         <platform>multi_platform_fedora</platform>
+{{% endif %}}
```

Comment:
these conditionals are not necessary. you can just say that it's applicable to all three

Reply:
My concern is that we will end up generating definitions applicable for RHEL6, with checks for `augenrules` when it is not supported. These definitions will be in OVAL file for RHEL7 and Fedora.

Do you think it can cause problems or confusion?
Like why a check that is applicable for RHEL6 differ in OVAL files for different systems?

Reply:
Wait, that's not how it works though, is it? It will generate the rhel6 definitions with rhel6 contents and the rhel7 definitions for rhel7. You don't have to put conditionals around affected platform. The definitions inside rhel7 oval will be for rhel7...

Reply:
> It will generate the rhel6 definitions with rhel6 contents and the rhel7 definitions for rhel7.

Yes, but the definition in rhel7 oval will say in its affected/platform that it can be applied to rhel6, and this definition will check for augenrules, which rhel6 does not support.
While in rhel6 oval it will apply to rhel7, and not check rules defined for augenrules.

The scenario that someone will actually compare these definitions between oval files can be a very corner case though.

Reply:
Yes but that is inevitable and is even worse with multi_platform_all which we use a lot. I want to fix it in #3006 and would appreciate your input there.